mu_repo/action_open_url.py

```diff
@@ -73,17 +71,36 @@ def Run(params):
 
     repos_with_changes = set(ComputeReposWithChangesFromCurrentBranchToOrigin(
         repos_and_curr_branch, params, target_branch=dest))
-
-
+    # todo: identify the remote for the current branch
+    # get from the confif the remote.{REMOTE}.url
+    # parse it. XXX 
     import webbrowser
-    for repo, branch in repos_and_curr_branch:
+    #print(repos_and_curr_branch)
+    for repo, branch in repos_and_curr_branch:        
         keywords['source'] = branch
+        print(repo)
         if repo in repos_with_changes:
             import os.path
             if repo == '.':
                 repo = os.path.basename(os.path.realpath('.'))
             else:
                 repo = repo.replace('.', '').replace('/', '').replace('\\', '')
+            if '{r_name}' in pattern:
```

Comment:
Can you update the docs in this method to reflect the usages of `{r_name}`.

Also, since this command is now becoming a bit more complex, can you create a unit-test for it? 

... the test could mock `webbrowser.open_new_tab` to check if the url expected is the correct one when `{r_name}` is present.

Reply:
ok